### DISCUSSION TITLE
Releasing 1.9.3 on npm

### DISCUSSION THREAD

**Discussion by Zenulous** [prabhuignoto/react-chrono#142] [Q&A]
@prabhuignoto I would really appreciate it if you could release 1.9.3 on npm. This release contains the property `  useReadMore?: boolean;` in `TimelineModel.ts` which is not present in the latest release on npm (1.9.2). I happen to need this so it would be great to have it without needing to patch the npm module :)

**Comment by prabhuignoto** [comment]
Apologies for the delayed response. `useReadMore` is also available with the latest version 

https://www.npmjs.com/package/react-chrono
